### PR TITLE
Add perf test for url-fetch

### PR DESCRIPTION
**Pull request by kalman**

### CODE REVIEW

**Comment by kalman** [comment]
@ehalpern ptal

File: go/perf/suite/suite.go [hunk 1]
```diff
@@ -338,29 +346,66 @@ func (suite *PerfSuite) NewAssert() *assert.Assertions {
...
 	fn()
 	suite.paused += time.Since(start)
 }
 
+// OpenGlob opens the concatenation of all files that match pattern, returned
+// as []io.Reader so it can be used immediately with io.MultiReader.
+//
+// Large CSV files in testdata are broken up into foo.a, foo.b, etc to get
+// around GitHub file size restrictions.
+func (s *PerfSuite) OpenGlob(pattern ...string) []io.Reader {
```
Comment (kalman) [inline go/perf/suite/suite.go]: These haven't changed, just moved verbatim from csv-import to here.

File: go/perf/suite/suite.go [hunk 2]
```diff
@@ -338,29 +346,66 @@ func (suite *PerfSuite) NewAssert() *assert.Assertions {
 	return assert.New(suite.T)
 }
 
-// TempFile creates a temporary file, which will be automatically cleaned up by the perf test suite.
-func (suite *PerfSuite) TempFile(prefix string) *os.File {
-	f, err := ioutil.TempFile("", prefix)
+// TempFile creates a temporary file, which will be automatically cleaned up by
+// the perf test suite. Files will be prefixed with the test's dataset ID
+func (suite *PerfSuite) TempFile() *os.File {
```
Comment (kalman) [inline go/perf/suite/suite.go]: Nicer API

File: go/perf/suite/suite_test.go [hunk 3]
```diff
@@ -24,25 +24,57 @@ type testSuite struct {
 	foo, bar, abc, def, nothing, testimate int
 }
 
-// This is the only test that does anything interesting. The others are just to test naming.
-func (s *testSuite) TestInterestingStuff() {
```
Comment (kalman) [inline go/perf/suite/suite_test.go]: Lazy me for not splitting these up in the first place

**Comment by kalman** [comment]
jenkins: perf this

**Comment by kalman** [comment]
jenkins: perf this

**Comment by ehalpern** [comment]
lgtm

**Pull request merged**